models/demos/wormhole/convnet_mnist/tt/convnet_mnist.py

```diff
@@ -0,0 +1,172 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import ttnn
+import torch.nn.functional as F
+from torch import nn
+
+
+def convnet_mnist(
+    input_tensor,
+    parameters,
+    device,
+    mesh_mapper,
+    mesh_composer,
+):
+    batch_size = input_tensor.shape[0]
+    torch_maxpool = True
+
+    conv_config = ttnn.Conv2dConfig(
+        dtype=ttnn.bfloat16,
+        weights_dtype=ttnn.bfloat16,
+        math_fidelity=ttnn.MathFidelity.LoFi,
+        activation="",
+        shard_layout=ttnn.TensorMemoryLayout.HEIGHT_SHARDED,
```

Comment:
Don't set shard_layout. We have a new feature to automatically choose the best shard layout.

Reply:
Sure @sankarmanoj-tt , will change the config and update the commit.